hive-tables-creation.py

```diff
@@ -0,0 +1,44 @@
+import boto3
+
+client = boto3.client('glue')
+response = client.create_table(
+    DatabaseName='analytical_dataset_generation',
+    TableInput={
+        'Name': 'core_contract_adg',
+        'Description': 'core_contract_adg',
```

Comment:
Might be worth standardising on a common description for these tables to mark where they get populated from? Something like "DB: core, Collection: contract: Topic: db.core.contract" perhaps?

Reply:
Modified as suggested

Reply:
Modified as suggested